components/Article.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+	<div>
+		<h1 class="title">
+			<b>{{ Title }}</b>
+		</h1>
+		<p id="api-content" v-html="Content" />
+	</div>
+</template>
+
+<script>
+export default {
+	props: {
+		Title: {
+			type: String,
+			default: "Untitled Post"
+		},
+		Content: {
+			type: String,
+			default: "We Count content"
```

Comment:
What would be a better option for communicating that a page or post has no content?